pkg/flow/flow_mem_driver.go

```diff
@@ -2089,6 +2179,15 @@ func (fc *FlowCollector) reconcileRecords() error {
 		}
 	}
 
+	for _, source := range fc.eventSources {
+		t := time.Now()
+		diff := uint64(t.UnixNano())/uint64(time.Microsecond) - source.EventSourceRecord.LastHeard
+		if diff > 30*oneSecond {
```

Comment:
How do you feel about 30 seconds? Do you think using the same value defined
through `--flow-collector-record-ttl` is a bit too much?

Reply:
record-ttl is 15 minutes default and a number of users have commented that they expect what is rendered in the console to be "real time". For example, if a site is deleted, they expect it to be removed soon after. It seems like anything more than 30-60 seconds would lose the real time feel. Additional user response should help with tuning this behavior.

Reply:
Fine. I believe it is ok to go with it now and address later based on users feedback.